### PR TITLE
fix: imagery-import-cogify use Basemaps PAT TDE-953

### PR DESCRIPTION
**Pull request by amfage**
#### Motivation

The new cluster does not contain all the old github ssh keys so the Basemaps workflows that are using them need to move to the Basemaps PAT which does exist on the new cluster.

#### Modification

Remove the old secret reference in imagery-import-cogify and use the Basemaps PAT instead.
Remove the imagery-import workflow that was using the old github ssh key as this workflow has been superseded by imagery-import-cogify.
Update the standardising-publish-import workflow that was still using the old imagery-import workflow.

#### Checklist

_If not applicable, provide explanation of why._

- [ ] Tests updated
- [ ] Docs updated
- [X] Issue linked in Title

### CODE REVIEW

File: workflows/imagery/standardising-publish-import.yaml [hunk 1]
```diff
@@ -41,8 +41,12 @@ spec:
         value: 'true'
       - name: retile
         value: 'false'
+      - name: version_argo_tasks
```
Comment (blacha) [inline workflows/imagery/standardising-publish-import.yaml:44]: we now have two versions for argo tasks?

Comment (amfage) [reply workflows/imagery/standardising-publish-import.yaml:44]: One has underscores and the other dashes! It is not ideal. Were we going to move everything to underscores at some point?

Comment (blacha) [reply workflows/imagery/standardising-publish-import.yaml:44]: underscores would be best, saves us doing this `['foo-bar']` vs `.foo_bar`

Comment (amfage) [reply workflows/imagery/standardising-publish-import.yaml:44]: Maybe not a change for this PR, though.

Comment (blacha) [reply workflows/imagery/standardising-publish-import.yaml:44]: I don't seem them used in this file though?

Comment (amfage) [reply workflows/imagery/standardising-publish-import.yaml:44]: It's for the downstream WorkflowTemplate `basemaps-imagery-import-cogify`

**Comment by amfage** [comment]
Out of date, will replace when `github-linz-basemaps-pat` is renamed to `github-linz-li-bot-pat`

**Pull request closed without merging**